Sources/CacheAdvance/CacheHeaderHandle.swift

```diff
@@ -142,23 +144,27 @@ final class CacheHeaderHandle {
         return try handle.readDataUp(toLength: Int(FileHeader.expectedEndOfHeaderInFile))
     }
 
-    /// Checks if the expected header version matches the persisted header version.
+    /// Checks that the file's persisted metadata has the expected values.
     ///
     /// - Parameter persistedMetadata: The persisted header metadata.
-    /// - Returns: `true` if this object's header version matches that of `fileHeader`; otherwise `false`.
-    private func canOpenFile(with persistedMetadata: Metadata) -> Bool {
+    /// - Throws: `CacheAdvanceError.incompatibleHeader` if this object's header version does not match that of `fileHeader`.
```

Comment:
@dfed I committed the change I proposed here since it seemed like you agreed.

Reply:
Yes! Thank you for that 🙂